network_simulator/plot_tool.py

```diff
@@ -0,0 +1,300 @@
+import matplotlib.pyplot as plt
+import numpy as np
+
+from statistics import *
```

Comment:
Please don't use "from X import *" format in any project files anymore, as it can create nasty circular dependencies. Just import the things you need (check for errors in PyCharm). I have already fixed this in the team-debug branch, so you can just change this in the files you've written.